src/components/Discussion/Internal/Comment/Header.js

```diff
@@ -268,7 +269,11 @@ export const Header = ({ t, comment, menu, isExpanded, onToggle }) => {
           {isExpanded ? <IcCollapse /> : <IcExpand />}
         </button>
       )}
-      {menu && <CalloutMenu align='right'>{menu}</CalloutMenu>}
+      {menu && (
+        <CalloutMenu Element={props => <MoreIcon {...props} />} align='right'>
```

Comment:
```suggestion
        <CalloutMenu Element={MoreIcon} align='right'>
```
This pattern should be avoided. And seems unnecessary here.

`props => <MoreIcon {...props} />` creates a new function on every render. React uses function identity internally and if it is constantly changing it will cause unnecessary complex DOM re-building.

If it were necessary the component should be defined outside of the render method:
```
const MoreIconWithProp = props => <MoreIcon size={24} {...props} />
const MyComponent = () => {
  return <CalloutMenu Element={MoreIconWithProp} />
}
const MyVeryDynamicComponent = ({size}) => {
  const MoreIconWithDynamicProp = useMemo(() => props => <MoreIcon size={size} {...props} />, [size])
  return <CalloutMenu Element={MoreIconWithDynamicProp} />
}
```